kafka-rest/src/test/java/io/confluent/kafkarest/controllers/TopicConfigManagerImplTest.java

```diff
@@ -99,6 +103,39 @@
           /* isSensitive= */ true,
           ConfigSource.UNKNOWN,
           /* synonyms= */ emptyList());
+  private static final TopicConfig ALT_TOPIC_CONFIG_1 =
```

Comment:
nit: Here and below, I'd even suggest just `ALT_CONFIG_1` and so on, or `CONFIG_4` and so on, as it's a shorter name without being less informative than the `CONFIG_1`/`CONFIG_2`/`CONFIG_3` that we're already using.